src/StackExchange.Redis/EndPointCollection.cs

```diff
@@ -208,12 +209,12 @@ internal async Task ResolveEndPointsAsync(ConnectionMultiplexer multiplexer, Log
                         }
                         else
                         {
-                            log?.WriteLine($"Using DNS to resolve '{dns.Host}'...");
+                            log?.LogInformation($"Using DNS to resolve '{dns.Host}'...");
                             var ips = await Dns.GetHostAddressesAsync(dns.Host).ObserveErrors().ForAwait();
                             if (ips.Length == 1)
                             {
                                 ip = ips[0];
-                                log?.WriteLine($"'{dns.Host}' => {ip}");
+                                log?.LogInformation($"'{dns.Host}' => {ip}");
```

Comment:
Seems like a lot of these logging levels should have been LogDebug or LogTrace

Reply:
Based on what? This is a core part of connecting to the server, knowing which IP we connected to is definitely information level in my mind. I'm open to adjusting these of course, but don't agree with the specific thing commented on here. I think most people would default to only-errors for StackExchange.Redis, but anything beneath that like connecting when we're a client with a persistent connection is, IMO, information level.

Reply:
I agree it could be important and can see if your only logging / interaction is with this library you might prefer it. However, most apps don't need this kind of logging and just adds to the noise. as the default log level is info 

But if you are having connectivity issues you are probably going down to the debug or trace levels anyways. Based on default out of the box log levels. Also, for most people this information is not needed and might even be a possible security concern leaking information about services (log collection tends to have different roles and policies).